tests/jsthemis/test.js

```diff
@@ -175,6 +175,28 @@ describe("jsthemis", function(){
 
 describe("jsthemis", function(){
     describe("secure cell", function(){
+        describe("key generation", function(){
+            const defaultLength = 32
+            it("generates new key buffer", function(){
+                var masterKey = new addon.SymmetricKey()
+                assert.equal(masterKey.length, defaultLength)
```

Comment:
actually here not checked that generated new key buffer. we should compare references/values to be sure, not that same length was used.

Reply:
Mmm... I’d say this is a bit excessive. But sure, we can add

```javascript
var key1 = new addon.SymmetricKey()
var key2 = new addon.SymmetricKey()
assert.notDeepEqual(key1, key2) // checks that content is not the same
assert.notEqual(key1, key2) // checks that objects are distinct
```

Though I doubt that this test will prevent any coding mistakes.

Reply:
I like to add it because we should be sure that our new layer of js code correctly uses themis core. If it was simple proxying calls like in c++/golang then it will be really overhead. But in wrappers like jsthemis/phpthemis where we deal with language wrapper API we should do more checks IMHO because we have more places for mistakes.

Reply:
Okay, now I get your point. That’s a pretty valid concern. Thank you for the explanation.